consul/docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: '2'
+services:
+  consul:
+    image: "consul:0.8.0"
+    ports:
+      - "8500:8500"
+  audit:
```

Comment:
nitpicky, but I find it easier to read docker-compose files when the service definitions are separated by an extra newline.